Index: project/scalatest.scala
===================================================================
--- project/scalatest.scala	(revision 4677)
+++ project/scalatest.scala	(working copy)
@@ -27,10 +27,12 @@
                               "org.scalatest.PackageAccessConstructorSuite")
                               
    lazy val root = Project("scalatest", file(".")) settings(
+     scalaHome in ThisBuild := Some(file("/Users/luc/scala/scala-effects/build/pack")),
+     unmanagedBase := file("/Users/luc/scala/scala-effects/build/pack"),
      organization := "org.scalatest",
      version := "1.9-2.10.0-M6-B2",
-     scalaVersion := "2.10.0-M6",
-     libraryDependencies ++= simpledependencies,
+     scalaVersion := "scala-effects",
+     libraryDependencies := simpledependencies,
      resolvers += "Sonatype Public" at "https://oss.sonatype.org/content/groups/public",
      sourceGenerators in Compile <+= 
          (baseDirectory, sourceManaged in Compile) map genFiles("gen-main", "GenGen.scala")(GenGen.genMain),
@@ -60,7 +62,7 @@
    }
 
    def simpledependencies = Seq(
-     "org.scala-lang" % "scala-actors" % "2.10.0-M6",
+     // "org.scala-lang" % "scala-actors" % "2.10.0-M6",
      "org.scala-tools.testing" % "test-interface" % "0.5",  // TODO optional
      "org.scalacheck" % "scalacheck_2.10.0-M6" % "1.10.0",   // TODO optional
      "org.easymock" % "easymockclassextension" % "3.1",   // TODO optional
